src/microbe_stage/Microbe.Interior.cs

```diff
@@ -488,8 +519,32 @@ public float EjectCompound(Compound compound, float maxAmount)
         return result;
     }
 
+    public Dictionary<Compound, float> CalculateAdditionalDigestibleCompounds()
+    {
+        var result = new Dictionary<Compound, float>();
+
+        // Add some part of the build cost of all the organelles
+        foreach (var organelle in organelles!)
+        {
+            foreach (var entry in organelle.Definition.InitialComposition)
+            {
+                var existing = 0.0f;
+
+                if (result.ContainsKey(entry.Key))
```

Comment:
TryGetValue?

Reply:
Most of these I just ripped out of the already existing code but anyway can a similar way be done here https://github.com/Revolutionary-Games/Thrive/pull/3106#discussion_r913128130?

Reply:
Yes. 
Sorry for that existing code, I wrote a bunch of it before I realized that's how dictionaries work.

Reply:
Fixed.